Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM openjdk
+WORKDIR /
+ADD target/conter-0.0.1-SNAPSHOT.jar conter-0.0.1-SNAPSHOT.jar
```

Comment:
Please make a separate PR (or I can, if you don't get to it by Monday), to be merged before this one, correcting the spelling of the artifact name.